backends/github/labels/github-labels.js

```diff
@@ -0,0 +1,127 @@
+import GithubAPI from "../api/github-api.js";
+
+/**
+ * Backend for performing operations with labels on GitHub repositories.
+ * @category GitHub
+ */
+export default class GithubLabels extends GithubAPI {
+	static urls = [
+		{hostname: "api.github.com", pathname: "repos/:owner/:repo/labels"},
+		{hostname: "api.github.com", pathname: "/repos/:owner/:repo/labels"},
+		{hostname: "api.github.com", pathname: "/repos/:owner/:repo/issues/:issue_number/labels"},
+		{hostname: "api.github.com", pathname: "/repos/:owner/:repo/milestones/:milestone_number/labels"},
+		{hostname: "github.com", pathname: "/:owner/:repo/labels"},
```

Comment:
We also have
```js
{hostname: "github.com", pathname: "/:owner/:repo/issues/labels"},
{hostname: "github.com", pathname: "/:owner/:repo/milestones"},
```
but they don't work without an issue (milestone) number respectively.

Reply:
Where? I don't see them above!

Reply:
I added them initially but later deleted them. That's why they are not there anymore. In the previous comment, I should've said, “We might also have... but they don't work without an issue (milestone) number respectively.“ 😅

Reply:
Indeed, we might pass something like `https://github.com/mavoweb/mavo/issues/1021/labels` to `Backend.from()`. If we add those two patterns to the list of supported URLs, we'll get a list of labels for issue #1021. This works because, under the hood, we perform requests using host `api.github.com`, not `github.com`.

But this URL (and all similar) itself returns `404`. And if we think that URLs starting with `github.com` will be copied from the address bar when visiting the corresponding page, it's a bit doubtful that those URLs will be copied whenever. I meant this when I talked about how they don't work.

OTOH, if we suppose that the author generates URLs they want to use with the backend dynamically, they will probably use the `github.com` as a host. In that case, it's worth adding them as supported URLs. I checked, and they work fine.

Reply:
I have an idea. What if we, instead of existing URL patterns, use the following ones?

```js
{hostname: "(api.)?github.com", pathname: "(/repos)?/:owner/:repo/labels"},
{hostname: "(api.)?github.com", pathname: "(/repos)?/:owner/:repo/issues/:issue_number/labels"},
{hostname: "(api.)?github.com", pathname: "(/repos)?/:owner/:repo/milestones/:milestone_number/labels"},
```

It will allow authors to use the backend without hassle with all supported URLs, regardless of the host and whether it returns `404` or not.